osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -79,7 +79,7 @@ public void ShowUser(User user, bool fetchOnline = true)
         {
             Show();
```

Comment:
What about moving this behind the check. Then it shouldn't open at all. Opening empty profiles feels really weird anyways.